stores/src/main/java/shop/category/Category.java

```diff
@@ -0,0 +1,27 @@
+package shop.category;
+
+import shop.Product;
+
+import java.util.List;
+
+public abstract class Category {
+
+    protected List<Product> productList;
+    protected String name;
```

Comment:
your impl depends on humans attention, not to  forget to call this.name = .... Please think how this behavior could be standardized and be required (constructor)